jinahub/indexers/searcher/compound/NumpyPostgresSearcher/__init__.py

```diff
@@ -34,9 +37,10 @@ def __init__(self, dump_path=None, **kwargs):
 
     @requests(on='/search')
     def search(self, docs: 'DocumentArray', parameters: Dict = None, **kwargs):
+        inner_parameters = parameters.get(self.metas.name, parameters)
```

Comment:
This is not needed, We decided that this will be implictly done. In this case u may miss parameters that are global

Reply:
I see. Fixed